spec/datetimeformat.html

```diff
@@ -105,6 +105,8 @@ <h1>InitializeDateTimeFormat ( _dateTimeFormat_, _locales_, _options_ )</h1>
           1. Let _prop_ be the name given in the Property column of the row.
           1. Let _value_ be ? GetOption(_options_, _prop_, `"string"`, &laquo; the strings given in the Values column of the row &raquo;, *undefined*).
           1. Set _opt_.[[&lt;_prop_&gt;]] to _value_.
+        1. If _opt_.[[Second]] is not *undefined*, then
```

Comment:
@littledan , @sffc @fabalbon  and everyone:
Do you think we should set _opt_.[[MillisecondDigits]] only if _opt_.[[Second]] is not *undefined* or it should be set unconditionally?

Reply:
I think we should set _opt._[[MillisecondDigits]] regardless. Like @HolgerJeromin commented, second can be set without hour. In InitializeDateTimeFormat,_opt_.[[Day]] will be set without _opt_.[[Month]].

Reply:
ok, I just change the PR, then the detection can be simplified as
```
function IsMillisecondDigitsOptionAvaiable() { 
  return (new Intl.DateTimeFormat()).resolvedOptions().millisecondDigits !== undefined; 
}
```

Reply:
Off topic (?): The detection code is surprising to me. Stable Chrome returns:

````js
(new Intl.DateTimeFormat()).resolvedOptions() // results in
{
	calendar: "gregory",
	day: "numeric",
	locale: "en-US",
	month: "numeric",
	numberingSystem: "latn",
	timeZone: "Europe/Berlin",
	year: "numeric"
}
````
Why does a Chrome with your patches adds `millisecondDigits`?

Reply:
> Off topic (?): The detection code is surprising to me. Stable Chrome returns:
> 
> ```js
> (new Intl.DateTimeFormat()).resolvedOptions() // results in
> {
> 	calendar: "gregory",
> 	day: "numeric",
> 	locale: "en-US",
> 	month: "numeric",
> 	numberingSystem: "latn",
> 	timeZone: "Europe/Berlin",
> 	year: "numeric"
> }
> ```
> 
> Why does a Chrome with your patches adds `millisecondDigits`?

Chrome read the option and convert them into "skeleton" and create LocalizedNumberFormatter.In the resolvedOptions it get the skeleton from the LocalizedNumberFormatter back and parse the skeleton to determine the options.
for the case of millisecondDigits, it decide to output it not not (which then will be undefined) purely based on the command flag which we control to turn on / off the feature. So "why  adds `millisecondDigits`" will only based on the command line flag but the value it output will based on the value of the skeleton.

Reply:
> Off topic (?): The detection code is surprising to me. Stable Chrome returns:
> 
> ```js
> (new Intl.DateTimeFormat()).resolvedOptions() // results in
> {
> 	calendar: "gregory",
> 	day: "numeric",
> 	locale: "en-US",
> 	month: "numeric",
> 	numberingSystem: "latn",
> 	timeZone: "Europe/Berlin",
> 	year: "numeric"
> }
> ```
> 
> Why does a Chrome with your patches adds `millisecondDigits`?

not sure your question is asking how my patch in chrome work (so I can answer you by walking you through the C++ code) or how does the spec change work (so I can answer you by walking you through the changed spec).